hack/generate/samples/internal/go/v2/memcached_with_webhooks.go

```diff
@@ -326,8 +326,6 @@ const reconcileFragment = `// Fetch the Memcached instance
 			log.Error(err, "Failed to update Deployment", "Deployment.Namespace", found.Namespace, "Deployment.Name", found.Name)
 			return ctrl.Result{}, err
 		}
-		// Spec updated - return and requeue
-		return ctrl.Result{Requeue: true}, nil
```

Comment:
don't you still want to return here?

Reply:
(and the other spots you're removing below)

Reply:
We are only removing the re-queue true ONLY after the update. This PR is changing the mock data used to generate the samples and consequently the samples. 

See that in the example we need to requeu true before to check the spec.size in order to have the Deployment. However, after we succefully update it is not required to re-triger the reconcile.

Reply:
> We are ONLY removing the re-queue true after the update (it is only one spot place). 

I understand, i realize this is the same code duplicated in various places and you're making the same change in all those places.  I was just saying that I think all the places where you touched it, you should still be returning (just with requeue=false)


there's no reason to continue down to the logic that updates the pod names, after you have updated the replica count, because the pod names are going to be incorrect as soon as the replica count change takes effect.

So i still don't understand why you don't return here, and then when you get the next event (one telling you the deployment replica count was updated), you can reconcile the pod names.

Reply:
To update the CR status with the pod names we use first the client to List all Pods. See; https://github.com/operator-framework/operator-sdk/blob/ef24b823bd55d996b7afda82c3ee3ec25969fecf/hack/generate/samples/internal/go/v2/memcached_with_webhooks.go#L333-L342

So, why in your mind is required we re-start the reconcile after the client.update occurs successfully?

Reply:
We spoke about that. The problem is that might the pods are not created yet. Then, we decide to use the re-queue after X and clarifies why.  I will do that.